core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1756,3 +1744,143 @@ secret_configs:
         namespace: test-credentials
         name: registry-push-credentials-ci-central
         type: kubernetes.io/dockerconfigjson
+  # This secret contains the pull secrets for api.ci cluster and the local registry.
+  # It is used to pull images
+  #  * by image client: Before the registry-replacer (https://github.com/openshift/ci-tools/tree/master/cmd/registry-replacer)
+  #    replaces the direct reference of the images in api.ci,
+  #    OpenShift uses the pull secrets either in openshift-config/pull-secret (the global one) or the ones
+  #    with kubernetes.io/dockerconfigjson type in current namespace to pull images from the remote registry.
+  #    The global one is maintained by cluster admins and it is not ensured to contain a valid auth entry for api.ci,
+  #    so we maintain this one in the ci namespace and ci-operator will copy it over to the testing namespace.
+  #  * by the build pods created by ci-operator: https://bugzilla.redhat.com/show_bug.cgi?id=1861546
+  #  * (only apply to api.ci) by image client: for the image jobs generated with releases stanza, we need to access the images
+  #    from the release payload and they are in quay.io/openshift-release-dev/...
+  - from:
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: quay.io
+          registry_url: quay.io
+          auth_bw_attachment: pull-token
+        - bw_item: build_farm
+          registry_url: docker-registry.default.svc:5000
+          auth_bw_attachment: token_image-puller_ci_reg_auth_value.txt
```

Comment:
nit: all of these are `token_<sa>_<cluster>_auth_value.txt` but for `api.ci` we just have `ci` for the cluster name - why?

Reply:
Historically, it is called that way.
Like my other bad naming conventions.
It is on the BW for existing BW items.

They will be removed after api.ci is gone.